doc/user_guide.md

```diff
@@ -0,0 +1,1518 @@
+# Fritz User Guide
+
+## Alert filters in `Fritz`
+
+In this tutorial, we will discuss the alert filters in `Fritz`:
+- Technical details on the implementation
+- Filter examples with step-by-step explanations
+
+### Introduction
+
+One of `Fritz`'s submodules, `Kowalski`, constantly listens to the alert streams and persists the alert data in a 
+database. `Kowalski` additionally computes a number of frequently-used quantities such as the Galactic coordinates for 
+each alert, cross-matches them with several external catalogs, and executes machine learning models. 
+
+`Kowalski` uses `MongoDB`, a document-based NoSQL database, on the backend. 
+See [here](https://github.com/dmitryduev/ay119/blob/master/databases/mongodb.ipynb) and the references therein 
+for a brief introduction into `MongoDB`.   
+
+#### Filtering implementation
```

Comment:
This section is somewhat technical. Rather than having it be the first thing the users see after the intro section, I'd recommend putting a quick-start section here, showing a toy filter that is complex enough to communicate some key ideas but not so complicated that users are overwhelmed with new features -- something that is < 30 LOC